x-pack/solutions/search/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/pipelines/pipelines.tsx

```diff
@@ -65,6 +66,29 @@ export const SearchIndexPipelines: React.FC = () => {
   const { makeRequest: revertPipeline } = useActions(RevertConnectorPipelineApilogic);
   const apiIndex = isApiIndex(index);
   const extractionDisabled = getContentExtractionDisabled(index);
+  const [isRevertPipeline, setRevertPipeline] = useState(false);
+  const buttonRef = useRef<HTMLButtonElement | null>(null);
+
+  useEffect(() => {
+    if (!isDeleteModalOpen) {
+      if (buttonRef.current) {
+        buttonRef.current.focus();
+      }
+      if (isRevertPipeline) {
+        const pipelinesButton = document.querySelector<HTMLDivElement>(
+          `[id="${SearchIndexTabId.PIPELINES}"]`
+        );
+        if (pipelinesButton) {
+          pipelinesButton.focus();
+        }
+      }
+    }
```

Comment:
Should we handle the buttonRef case in an else? 
```suggestion      
      if (isRevertPipeline) {
        const pipelinesButton = document.querySelector<HTMLDivElement>(
          `[id="${SearchIndexTabId.PIPELINES}"]`
        );
        if (pipelinesButton) {
          pipelinesButton.focus();
        }
      }
    } else if (buttonRef.current) {
      buttonRef.current.focus();
    }
```

Not sure if there is a case where we would end up calling `focus()` on 2 elements in this handler.